### PR TITLE
docs: add changelog v1.9.0

### PR DESCRIPTION
**Pull request by adela-bytebase**
add changelog v.1.9.0

### CODE REVIEW

**Review by boojack — APPROVED** [review]
LGTM

File: content/changelog/bytebase-1-9-0.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## 🎠 Community
+
+- Support managing with Supabase databases: [https://bytebase.com/docs/how-to/integrations/supabase](/docs/how-to/integrations/supabase)
+
+## 📰 Fresh Off the Press
+
+- [How to Manage Database Access Control](/blog/how-to-manage-database-access-control)
+- [MotherDuck, from SQLite to the Docker for Data](/blog/motherduck-from-sqlite-to-the-docker-for-data)
+- [How to Synchronize Database Schemas](/blog/how-to-synchronize-database-schemas)
+- [How schema sync works in Bytebase](/blog/how-schema-sync-work)
```
Comment (tianzhou) [inline content/changelog/bytebase-1-9-0.md:36]:
```suggestion
- [How Schema Sync Works in Bytebase](/blog/how-schema-sync-work)
```

Comment (tianzhou) [reply content/changelog/bytebase-1-9-0.md:36]: I will change the title in the article in a follow PR

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**